README-win.md

```diff
@@ -0,0 +1,84 @@
+#netdisco-docker on windows10
+
+## requirements
+
+* windows 10 x64, version 1903
+  * nested virtualization needs to be enabled in the bios
+  * the following features need to be enabled in windows:
+    * containers
+    * hyper-v
+    * virtual machine platform
+    * windows hypervisor platform 
+* docker version 19.03.0-rc2
+  * docker must be set to use linux containers
+* docker-compose version 1.24.0
+
+## installing netdisco-docker on windows
+
+unlike linux we use local volumes on windows, this takes care of differences in
+userid/acl handling.
+
+* first create local volumes for netdisco's storage
```

Comment:
docker-compose will create volumes that don't exist. Everything should be defined by the docker-compose.yml file. This walk through should be able to be greatly simplified.

Reply:
i'll give this another try, but when i tried i couldn't get it to work. perhaps it'ds a windows specific thing.

Reply:
turns out all that's needed for that is defined the volumes as external: false
https://github.com/inphobia/netdisco-docker/commit/a90bfc3a6c8c63e8e83e3e4c68e5a52d04d45127